docs/zh-CN/sql-manual/sql-reference/Show-Statements/SHOW-USER.md

```diff
@@ -31,9 +31,19 @@ under the License.
 SHOW USER
 
 ### Description
+该语句用于展示当前登录用户的用户名
 
-### Example
+语法：
```

Comment:
Add an example result and explain each column in result set.

Reply:
@morningman updated code